apiserver/applicationoffers/applicationoffers.go

```diff
@@ -29,8 +28,7 @@ type environFromModelFunc func(string) (environs.Environ, error)
 // implementation of the api end point.
 type OffersAPI struct {
 	BaseAPI
-	dataDir    string
-	getEnviron environFromModelFunc
+	dataDir string
```

Comment:
It is a carry over from when there were 2 facades dealing with this functionality - one for managing offers, one for searching for offers. I kept it because the base api contains a lot of infrastructure/helper type code, whereas the offers api contains the core business logic of the apis themselves. It keeps the file sizes smaller and hopefully easier to read.